DataFormats/TauReco/src/classes_def_2.xml

```diff
@@ -156,8 +156,6 @@ for(auto const& ref : onfile.selectedIsolationPFGammaCands_) {
 
 
 
-
-
```

Comment:
I think I did this incorrectly. I'll try again.

On Fri, Mar 6, 2015 at 6:04 PM, Slava Krutelyov <notifications@github.com>
wrote:

> In DataFormats/TauReco/src/classes_def_2.xml
> <https://github.com/cms-sw/cmssw/pull/8119#discussion_r25959275>:
>
> > @@ -156,8 +156,6 @@ for(auto const& ref : onfile.selectedIsolationPFGammaCands_) {
> >
> >
> >
> > -
> > -
>
> what happened here? why an empty change?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8119/files#r25959275>.
>